index.js

```diff
@@ -11,10 +11,12 @@ AFRAME.registerComponent('altspace', {
    * usePixelScale will allow you to use A-Frame units as CSS pixels.
    * This is the default behavior for three.js apps, but not for A-Frame apps.
    * verticalAlign puts the origin at the bottom, middle (default), or top of the Altspace enclosure.
+   * disableFor2D prevents the scene from being created if enclosure is flat.
    */
   schema: {
     usePixelScale: { type: 'boolean', default: 'false'},
-    verticalAlign: { type: 'string', default: 'middle'}
+    verticalAlign: { type: 'string',  default: 'middle'},
+    disableFor2D:  { type: 'boolean', default: 'true'}
```

Comment:
Not sure if `disableFor2D` is the best name. It's probably best if we reinforce our terminology instead of mentioning "2d enclosures" or "flat enclosures". Maybe `disableInDisplays` would be better. Also, setting this to try by default is technically a breaking change. I would set it to false by default.